CONTRIBUTING.md

```diff
@@ -8,14 +8,13 @@ You can improve anything you feel like or [solve issues](https://github.com/Hars
 
 ## Local setup
 
-> The following commands are for Windows. Please modify them for other Operating Systems.
```

Comment:
Can you please let this line be there?
MacOS needs `python3` in its commands to use Python 3.

Reply:
There is only one instance of `python`, so it might be better to add a line like (use `python3` for MacOS).

Reply:
Alright. I'll do it for `pip3` as well.